<launch>

  <!-- cascaded pid control node -->
  <node pkg="cascaded_pid_control" type="cascaded_pid_control" name="CascadedPidControl" output="screen">
    <remap from="~odometry" to="/CheatOdometryNode/odometry" />
    <remap from="~rateThrust" to="/uav/input/rateThrust" />

    <param name="~publish_debug_topic" value="True" />
  </node>

  <!-- cheat odometry node -->
  <node pkg="cascaded_pid_control" type="cheat_odometry_node" name="CheatOdometryNode" output="screen" />

</launch>